blocks/footer/footer.js

```diff
@@ -10,7 +10,7 @@ function createMobileMenu(block) {
     details.className = 'accordion-item';
     const summary = document.createElement('summary');
     summary.className = 'accordion-header';
-    summary.innerHTML = accordionItem.querySelector('strong').outerHTML;
+    summary.innerHTML = accordionItem.querySelector('strong') ? accordionItem.querySelector('strong').outerHTML : '';
```

Comment:
I think this is causing an anomaly on the struktur-level page. Sometimes the accordion is broken when I navigate there, and always if I refresh it's broken. The other club pages don't do that.
![image](https://github.com/aemsites/888de/assets/3883795/705e521f-6298-4851-8e62-9dd705c6de3f)
https://feat-clubsnav--888de--aemsites.hlx.page/888club/level-struktur

Reply:
@helms-charity refactored the code...looks like that resolved the issue.